packages/familie-backend/src/auth/tokenUtils.ts

```diff
@@ -87,5 +90,10 @@ export const hasValidAccessToken = (req: Request, key = tokenSetSelfId) => {
     if (!tokenSet) {
         return loggOgReturnerOmTokenErGyldig(req, key, false);
     }
-    return loggOgReturnerOmTokenErGyldig(req, key, new TokenSet(tokenSet).expired() === false);
+    return loggOgReturnerOmTokenErGyldig(req, key, erUtgått(tokenSet) === false);
 };
+
+// kallkjedene kan ta litt tid, og tokenet kan i corner-case gå ut i løpet av kjeden. Så innfører et buffer
+// på 2 minutter.
+const erUtgått = (tokenSet: any): boolean =>
```

Comment:
```suggestion
const erUtgått = (tokenSet: TokenSet): boolean =>
```
Bruke riktig type kanskje?

Reply:
metoden getTokenSetsFromSession returnerer berre tokenSet: any, så må konvertere til TokenSet på neste linje